cluster-operator/src/main/java/io/strimzi/operator/cluster/model/metrics/StrimziMetricsReporterConfig.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model.metrics;
+
+/**
+ * This class holds Strimzi Metrics Reporter and related configuration keys.
+ */
+public class StrimziMetricsReporterConfig {
```

Comment:
Why does this need a separate public class? Why not have it directly in the model class as we have for most other stuff in other classes?

Reply:
This will come in handy when we start to add the other components. If you think it's outside the scope of this PR, I can remove it until later on maybe?

Reply:
Once again - you might be right - but you have to explain what would be the value/use in the future.

Reply:
Thanks for the clarification. I will remove for now.